examples/SerialIface/SerialIface.ino

```diff
@@ -36,16 +42,20 @@
 #include <OPL2.h>
 
 // Text command mode
-#define STARTUP_MSG   "HLO!\n"
-#define BUF_SIZE_CMD  "BUF?\n"
+#define STARTUP_MSG       "HLO!\n"
+#define BUF_SIZE_CMD      "BUF?\n"
+#define FAST_BUF_SIZE_CMD "B0F?\n"
 
 // Binary command mode
-#define ACK_RSP         'k'
-#define RESET_CMD       0
-#define BINARY_CMD_SIZE 5
+#define ACK_RSP             'k'
+#define RESET_CMD            0
+#define BINARY_CMD_SIZE      5
+#define FAST_BINARY_CMD_SIZE 2
 
 OPL2 opl2;
-bool ready = false;
+
+// This function pointer will be set when in binary command mode.
+void (*processCmdFn)(void) = &waitForBufSizeCmd;
```

Comment:
Please keep in mind that this is example code that must be understandable for less experienced developers! Using a function pointer here does not give you any significant gain in performance.

Reply:
Fair enough, personally I don't really mind either way and haven't benchmarked it myself.